controllers/controllers.js

```diff
@@ -45,3 +46,11 @@ exports.getCommentsByArticle = ({ params: { article_id } }, res, next) => {
     })
     .catch((err) => next(err));
 };
+
+exports.postComment = ({ body, params: { article_id } }, res, next) => {
+  createComment(body, article_id)
+    .then((comment) => {
+      res.status(201).send({ created_comment: comment });
```

Comment:
for a bit of shorthand you can rename the `comments` variable to be `created_comment` and then send `{ created_comment }`